balboa-client-dispatcher/README.md

```diff
@@ -27,7 +27,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-dispatcher-client" %% "[0.15, )",
+   "com.socrata" % "balboa-dispatcher-client" %% "[0.16, )",
```

Comment:
Lets update #23 to update all the README to use the + notation for those who are not intuned with mathematical notation.